test/parallel/test-tls-alert.js

```diff
@@ -28,8 +28,8 @@ function loadPEM(n) {
 
 var server = tls.Server({
   secureProtocol: 'TLSv1_2_server_method',
-  key: loadPEM('agent2-key'),
-  cert:loadPEM('agent2-cert')
+  key:  loadPEM('agent2-key'),
```

Comment:
It didn't need to change, but it appeared to me that the inconsistent spacing of the original was an attempt to align the values in a single column, so I did that. Don't really care either way TBH.